docs/source/models.rst

```diff
@@ -163,6 +171,18 @@ The written ``conda.yaml`` file:
       - scikit-learn==0.23.2
       - cloudpickle==1.6.0
 
+The written ``python_env.yaml`` file:
+
+.. code-block:: yaml
+
+    python: 3.8.8
+    build_dependencies:
+    - pip==21.1.3
+    - setuptools==57.4.0
+    - wheel==0.37.0
+    dependencies:
+    - -r requirements.txt
```

Comment:
```suggestion
    python: 3.8.8
    build_dependencies:
      - pip==21.1.3
      - setuptools==57.4.0
      - wheel==0.37.0
    dependencies:
      - -r requirements.txt
```
does this render ok with yaml-style indentations?

Reply:
Yep!

Reply:
Fixed!